<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreatorMainWindow</class>
 <widget class="QMainWindow" name="CreatorMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Окно создания &quot;Японских рисунков&quot;</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Japuzzle.qrc">
    <normaloff>:/Icons/Creator.png</normaloff>:/Icons/Creator.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Minimum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>0</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Minimum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="1">
     <widget class="QWidget" name="widgetZoom" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="widgetResize" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>ширина:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBoxWidth"/>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>высота:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBoxHeight"/>
          </item>
          <item>
           <widget class="QToolButton" name="toolButtonResizeOk">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSliderZoom">
         <property name="minimum">
          <number>5</number>
         </property>
         <property name="maximum">
          <number>15</number>
         </property>
         <property name="pageStep">
          <number>1</number>
         </property>
         <property name="value">
          <number>10</number>
         </property>
         <property name="tracking">
          <bool>true</bool>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="invertedAppearance">
          <bool>false</bool>
         </property>
         <property name="invertedControls">
          <bool>false</bool>
         </property>
         <property name="tickPosition">
          <enum>QSlider::NoTicks</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelZoom">
         <property name="text">
          <string>100%</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButtonZoomOut">
         <property name="text">
          <string>...</string>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButtonZoomIn">
         <property name="text">
          <string>...</string>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="2">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Minimum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="1">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Minimum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>0</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="1">
     <widget class="QScrollArea" name="scrollAreaMain">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>784</width>
         <height>117</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <widget class="EditWidget" name="editWidget" native="true">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>40</y>
          <width>651</width>
          <height>351</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>35</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>Вид</string>
    </property>
    <addaction name="actionBrick1"/>
    <addaction name="actionBrick5"/>
    <addaction name="actionBrick9"/>
    <addaction name="separator"/>
    <addaction name="actionZoomIn"/>
    <addaction name="actionZoomOut"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Правка</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionSaveAs"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="separator"/>
   <addaction name="actionBrick1"/>
   <addaction name="actionBrick5"/>
   <addaction name="actionBrick9"/>
  </widget>
  <action name="actionBrick1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Japuzzle.qrc">
     <normaloff>:/Icons/Brick1.png</normaloff>:/Icons/Brick1.png</iconset>
   </property>
   <property name="text">
    <string>Рисовать в &amp;1 блок</string>
   </property>
   <property name="toolTip">
    <string>Рисовать в 1 блок</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="actionBrick5">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Japuzzle.qrc">
     <normaloff>:/Icons/Brick5.png</normaloff>:/Icons/Brick5.png</iconset>
   </property>
   <property name="text">
    <string>Рисовать в &amp;5 блоков</string>
   </property>
   <property name="toolTip">
    <string>Рисовать в 5 блоков</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
  <action name="actionBrick9">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Japuzzle.qrc">
     <normaloff>:/Icons/Brick9.png</normaloff>:/Icons/Brick9.png</iconset>
   </property>
   <property name="text">
    <string>Рисовать в &amp;9 блоков</string>
   </property>
   <property name="toolTip">
    <string>Рисовать в 9 блоков</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+3</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="Japuzzle.qrc">
     <normaloff>:/Icons/Undo.png</normaloff>:/Icons/Undo.png</iconset>
   </property>
   <property name="text">
    <string>Отменить действие</string>
   </property>
   <property name="toolTip">
    <string>Отменить действие</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="Japuzzle.qrc">
     <normaloff>:/Icons/Redo.png</normaloff>:/Icons/Redo.png</iconset>
   </property>
   <property name="text">
    <string>Повторить действие</string>
   </property>
   <property name="toolTip">
    <string>Повторить действие</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Выход</string>
   </property>
   <property name="toolTip">
    <string>Выход</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="Japuzzle.qrc">
     <normaloff>:/Icons/Open.png</normaloff>:/Icons/Open.png</iconset>
   </property>
   <property name="text">
    <string>Открыть рисунок</string>
   </property>
   <property name="toolTip">
    <string>Открыть рисунок</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="Japuzzle.qrc">
     <normaloff>:/Icons/Save.png</normaloff>:/Icons/Save.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить рисунок</string>
   </property>
   <property name="toolTip">
    <string>Сохранить рисунок</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="Japuzzle.qrc">
     <normaloff>:/Icons/New.png</normaloff>:/Icons/New.png</iconset>
   </property>
   <property name="text">
    <string>Создать рисунок</string>
   </property>
   <property name="toolTip">
    <string>Создать рисунок</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="icon">
    <iconset resource="Japuzzle.qrc">
     <normaloff>:/Icons/ZoomIn.png</normaloff>:/Icons/ZoomIn.png</iconset>
   </property>
   <property name="text">
    <string>Увеличить</string>
   </property>
   <property name="toolTip">
    <string>Увеличить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+=</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="icon">
    <iconset resource="Japuzzle.qrc">
     <normaloff>:/Icons/ZoomOut.png</normaloff>:/Icons/ZoomOut.png</iconset>
   </property>
   <property name="text">
    <string>Уменьшить</string>
   </property>
   <property name="toolTip">
    <string>Уменьшить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset resource="Japuzzle.qrc">
     <normaloff>:/Icons/SaveAs.png</normaloff>:/Icons/SaveAs.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить рисунок как ...</string>
   </property>
   <property name="toolTip">
    <string>Сохранить рисунок как ...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionResize">
   <property name="icon">
    <iconset resource="Japuzzle.qrc">
     <normaloff>:/Icons/Menu Ok.png</normaloff>:/Icons/Menu Ok.png</iconset>
   </property>
   <property name="text">
    <string>Изменить размеры рисунка</string>
   </property>
   <property name="toolTip">
    <string>Изменить размеры рисунка</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EditWidget</class>
   <extends>QWidget</extends>
   <header>EditWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Japuzzle.qrc"/>
 </resources>
 <connections/>
</ui>
